src/ScalarPotentials/PointTypes.jl

```diff
@@ -228,4 +231,95 @@ end
 
 Base.convert(T::Type{NamedTuple}, x::PointTypes) = T(x)
 
+"""
+    get_inactivelayer_indeces(vec::AbstractVector{UInt8})::Vector{Vector{Int}}
+
+Scans a 1D vector and returns index groups that surround a `0x00` and
+contain both `0x07` and `0x0f`, bounded by `0x0d`.
+"""

```

Comment:
Can you comment on what `0x07`, `0x0f` and `0x0d` would be?
And would this need to be redefined if we were to add more bits to the point types?